src/apollo.js

```diff
@@ -1,7 +1,7 @@
 import { ApolloClient, InMemoryCache } from '@apollo/client';
 
 const apolloClient = new ApolloClient({
-  uri: 'https://48p1r2roz4.sse.codesandbox.io', // TODO change here
+  uri: 'http://localhost:4001/gql', // TODO change here
```

Comment:
Nitpick - try to remove the comments or update them as needed.

Reply:
removed comments